sql/share/errmsg-utf8.txt

```diff
@@ -6559,800 +7233,1056 @@ ER_ACCESS_DENIED_NO_PASSWORD_ERROR 28000
+        chi "创建索引'%-.192s'所需的多于'innodb_online_alter_log_max_size'字节的修改日志。请再试一次"
+        eng "Creating index '%-.192s' required more than 'innodb_online_alter_log_max_size' bytes of modification log. Please try again"
 
 ER_UNKNOWN_ALTER_ALGORITHM
-  eng "Unknown ALGORITHM '%s'"
+        chi "未知算法'%s'"
+        eng "Unknown ALGORITHM '%s'"
 
 ER_UNKNOWN_ALTER_LOCK
-  eng "Unknown LOCK type '%s'"
+        chi "未知锁定类型'%s'"
+        eng "Unknown LOCK type '%s'"
 
 ER_MTS_CHANGE_MASTER_CANT_RUN_WITH_GAPS
-  eng "CHANGE MASTER cannot be executed when the slave was stopped with an error or killed in MTS mode. Consider using RESET SLAVE or START SLAVE UNTIL"
+        chi "当从站因为错误停止或以MTS模式终止时，不呢执行CHANGE MASTER。考虑使用RESET SLAVE或START SLAVE UNTIL"
+        eng "CHANGE MASTER cannot be executed when the slave was stopped with an error or killed in MTS mode. Consider using RESET SLAVE or START SLAVE UNTIL"
 
 ER_MTS_RECOVERY_FAILURE
-  eng "Cannot recover after SLAVE errored out in parallel execution mode. Additional error messages can be found in the MariaDB error log"
+        chi "从并行执行模式下的从站错误后无法恢复。可以在MariaDB错误日志中找到其他错误消息"
+        eng "Cannot recover after SLAVE errored out in parallel execution mode. Additional error messages can be found in the MariaDB error log"
 
 ER_MTS_RESET_WORKERS
-  eng "Cannot clean up worker info tables. Additional error messages can be found in the MariaDB error log"
+        chi "无法清理工作者信息表。可以在MariaDB错误日志中找到其他错误消息"
+        eng "Cannot clean up worker info tables. Additional error messages can be found in the MariaDB error log"
 
 ER_COL_COUNT_DOESNT_MATCH_CORRUPTED_V2
-  eng "Column count of %s.%s is wrong. Expected %d, found %d. The table is probably corrupted"
-  ger "Spaltenanzahl von %s.%s falsch. %d erwartet, aber %d gefunden. Tabelle ist wahrscheinlich beschädigt"
+        chi "列数为%s.%s是错误的。预期的%d，找到%d。表可能损坏了"
+        eng "Column count of %s.%s is wrong. Expected %d, found %d. The table is probably corrupted"
+        ger "Spaltenanzahl von %s.%s falsch. %d erwartet, aber %d gefunden. Tabelle ist wahrscheinlich beschädigt"
 
 ER_SLAVE_SILENT_RETRY_TRANSACTION
-  eng "Slave must silently retry current transaction"
+        chi "从站必须静默地重试当前事务"
+        eng "Slave must silently retry current transaction"
 
 ER_UNUSED_22
-  eng "You should never see it"
+        chi "你永远不应该看到它"
+        eng "You should never see it"
 
 ER_TABLE_SCHEMA_MISMATCH
-  eng "Schema mismatch (%s)"
+        chi "架构不匹配(%s)"
+        eng "Schema mismatch (%s)"
 
 ER_TABLE_IN_SYSTEM_TABLESPACE
-  eng "Table %-.192s in system tablespace"
+        chi "表%-.192s在系统表空间中"
+        eng "Table %-.192s in system tablespace"
 
 ER_IO_READ_ERROR
-  eng "IO Read error: (%lu, %s) %s"
+        chi "IO读取错误:(%lu，%s）%s"
+        eng "IO Read error: (%lu, %s) %s"
 
 ER_IO_WRITE_ERROR
-  eng "IO Write error: (%lu, %s) %s"
+        chi "IO写错错误:(%lu，%s）%s"
+        eng "IO Write error: (%lu, %s) %s"
```

Comment:
I fixed the messages to say （%lu,%s）on merge to 10.6 (fae0ccad6ec391bf85c39070f784589d48e11951)